### ISSUE TITLE
Wood Chopping Issue

### ISSUE THREAD

**Issue by Koizad** [DarkPacks/SevTech-Ages#4544]
## Bug Description

I go to place regular Oak on the wood chopping block and it disappears and doesn't want to be chopped by clicking once over and over again. Might just be a texture glitch?

### Did this behavior use to work in the previous version?

 This is my first time playing with this modpack

## Steps to Reproduce (for bugs)

1. Make Wood Chopping Block
2. Place it
3. Place Oak on the block, not above it.
4. Fails to load?

## Logs

* Client/Server Log:
* Crash Log: Didn't crash

## Client Information
* Modpack Version: 3.1.7
* Java Version:1.12.2
* Launcher Used: Forge 14.23.5.2854
* Memory Allocated:6k
* Server/LAN/Single Player: Singleplayer
* Optifine Installed: Optifine HD Ultra F5
* Shaders Enabled: DoctourDreads Ultra V02

## World Information
* Modpack Version world created in: Survival
* Additional Content Installed: None, just the modpack

**Comment by sam-kirby** [comment]
Does this issue persist without Optifine/shaders?

**Comment by Koizad** [comment]
Yes, I turned off Optifine/Shaders, however I did a little digging and its
an issue with Twitch Studio, OBS, and OBS streamlabs. The issue doesn't
persist once I stop recording and shut down that software.
